### DISCUSSION TITLE
Streamlink GUI param to add new chat to existing window

### DISCUSSION THREAD

**Discussion by iDump-ster** [Chatterino/chatterino2#2460] [Q&A] [answered]
Hey the changes are amazing! But specifically to use Chattarino chat option has made it so now Chattarino properly populates the users chat through the Streamlink GUI on the selected streamer. 
I use `--channels "{channel}"` atm, but is there a way to force a single instanc/ it to append to the existing window?

**Comment by leon-richardt** [comment, marked as answer]
See relevant discussion in https://github.com/streamlink/streamlink-twitch-gui/issues/607. TL;DR: It is currently not possible to do this in Chatterino. Feel free to [open an issue ](https://github.com/Chatterino/chatterino2/issues/new?assignees=&labels=enhancement&template=feature-suggestion.md&title=) for your feature suggestion.
